app/Services/ForwardChatworkService.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Webhook;
+use App\Enums\PayloadHistoryStatus;
+
+class ForwardChatworkService
+{
+    protected $webhook;
+    protected $params;
+    protected $payloadHistoryRepository;
+
+    public function __construct(Webhook $webhook, $params, $payloadHistoryRepository)
+    {
+        $this->webhook = $webhook;
+        $this->params = $params;
+        $this->payloadHistoryRepository = $payloadHistoryRepository;
+    }
+
+    public function call()
+    {
+        $payloads = $this->webhook->payloads;
+        $params = $this->params;
+        $filledPayloads = [];
+        foreach ($payloads as $payload) {
+            $conditions = $payload->conditions;
+            $isValid = true;
+            foreach ($conditions as $condition) {
+                // 'return $payloads->fieldName == "value";'
+                $expression = 'return ' . $condition->field . ' '
+                                        . $condition->operator . ' "'
+                                        . $condition->value . '";';
+                if (!eval($expression)) {
+                    $isValid = false;
+                    break;
+                }
+            }
+
+            if ($isValid) {
+                array_push($filledPayloads, $payload);
+            }
+        }
+
+        if ($filledPayloads) {
+            // mapping data from params request into payload content
+            $messages = [];
+            foreach ($filledPayloads as $payload) {
+                $message = $this->generateMessage($payload->content, $this->params);
+                array_push($messages, $message);
+            }
+
+            // send message to chatwork
+            dd($messages);
+        } else {
+            // make a new failed payload history record
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Services/ForwardChatworkService.php</br>**Line**: 58</br>No space found after comma in function call